### PR TITLE
[8.0][html_text] Excerpt generator.

### PR DESCRIPTION
**Pull request by yajo**
This module adds a technical utility to allow the developer to get an excerpt from any HTML chunk.

You can choose the maximum amount of words or characters if you want.

See the README and inline docstrings for further details. Example usage found in https://github.com/OCA/website/pull/163.

### CODE REVIEW

File: html_text/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Text from HTML field
+====================
+
+This module extracts text from any chunk of HTML, without HTML tags or
```
Comment (elicoidal) [inline html_text/README.rst]: This module extracts text from any chunk of HTML, without HTML tags or
attributes. You can chose either:
- to truncate the result by amount of words or characters.
- to append an ellipsis (or any character(s)) at the end of the result.

File: html_text/models/ir_fields_converter.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# © 2016 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from lxml import html
```
Comment (elicoidal) [inline html_text/models/ir_fields_converter.py]: Should you protect the import (Try...Except)?

Comment (yajo) [inline html_text/models/ir_fields_converter.py]: This is an Odoo's dependency, so it is supposed to be always installed.

File: html_text/README.rst (hunk 1 above)
Comment (yajo) [inline html_text/README.rst]: Fixed.

**Comment by yajo** [comment]
Could somebody please tag this as "needs review"?

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Tagged as needs review

File: html_text/README.rst (hunk 1 above)
Comment (pedrobaeza) [inline html_text/README.rst]: I think it's better to put "This module provides some technical modules that allow to extract ..."

**Comment by pedrobaeza** [comment]
Little comment, but :+1:

**Comment by atchuthan** [comment]
code reviewed :+1:

**Comment by yajo** [comment]
Fixed.

**Pull request merged**